apps/reference/docs/architecture.mdx

```diff
@@ -15,3 +15,25 @@ Supabase is not a 1-to-1 mapping of Firebase. While we are building many of the
 Our technological choices are quite different from Firebase. Everything we use is open source. Wherever possible, we use and support existing tools rather than developing from scratch.
 
 Most notably, we use Postgres rather than a NoSQL store. This choice was deliberate. We believe that no other database offers the scalability and functionality required to compete with Firebase.
+
+## Feature Status
+
+| Product                    | Feature                | Stage |
```

Comment:
should we add a column to link to the relevant docs (if available)?

Reply:
added